pkg/aggregator/rates_aggregator.go

```diff
@@ -14,16 +14,16 @@ type RateAggregatorInterface interface {
 
 type RateAggregator struct {
 	lastRate        map[string]*Rate
-	notifier        notifier.NotifierInterface
+	notifierChain   notifier.NotifierChain
 	currencyConfigs []config.CurrencyConfig
 }
 
 func NewRateAggregator(
-	notifier notifier.NotifierInterface,
+	notifierChain *notifier.NotifierChain,
```

Comment:
`RateAggreator` in here is aware of concrete `NotifierChain` structure. From aggregator point of view I think it does not need to be aware if it's a chain or regular notifier, right? If so, how about to declare in `aggregator` package (as interfaces are prefered to be declared in client packages in golang) some `Notifier` interface which would include all of the required methods? It could help with unit testing as well, correct?